src/types/types.ts

```diff
@@ -1584,6 +1584,48 @@ export interface SendPrivateTransactionOptions {
   fast: boolean;
 }
 
+export enum AssetType {
+  NATIVE = 'NATIVE',
+  ERC20 = 'ERC20',
+  ERC721 = 'ERC721',
```

Comment:
is there any way we can combine this with any of the existing enums?

Reply:
difference with `AssetTransfersCategory`

- lowercase for `AssetTransfersCategory` vs uppercase for `SimulateAssetType`
- `INTERNAL` and `EXTERNAL` for `AssetTransfersCategory` vs `NATIVE` for `SimulateAssetType`
- `SPECIALNFT` for `AssetTransfersCategory` vs `SPECIAL_NFT` for `SimulateAssetType`

We could 

1. have a type that omits `INTERNAL`, `EXTERNAL` and adds `NATIVE`
2. lowercase `ERC20`, `ERC721`, `ERC1555` and `SPECIAL_NFT` at Simulation API level
3. convert `SPECIAL_NFT` to `SPECIAL_NFT` at Simulation API level

Reply:
wdyt?

Reply:
@thebrianchen let me know thoughts 🎈

Reply:
closing the loop here - synced with API committee

- keeping enum as is - going forward enums will be macro cased (uppercase and separated with _)
- Transfers API v2 should update enums to uppercase
- Transfers API v2 should change `SPECIALNFT` to `SPECIAL_NFT`